src/sql/src/func.rs

```diff
@@ -1921,6 +1923,12 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "acosh" => Scalar {
             params!(Float64) => UnaryFunc::Acosh(func::Acosh) => Float64, 2466;
         },
+        "constant_time_eq_bytes" => Scalar {
+            params!(Bytes, Bytes) => mz_expr::func::constant_time_compare_bytes => Bool, oid::FUNC_CONSTANT_TIME_EQ_BYTES_OID;
+        },
+        "constant_time_eq_strings" => Scalar {
+            params!(String, String) => mz_expr::func::constant_time_compare_strings => Bool, oid::FUNC_CONSTANT_TIME_EQ_STRINGS_OID;
+        },
```

Comment:
```suggestion
        "constant_time_eq" => Scalar {
            params!(Bytes, Bytes) => BinaryFunc::ConstantTimeEqBytes => Bool, oid::FUNC_CONSTANT_TIME_EQ_BYTES_OID;
            params!(String, String) => BinaryFunc::ConstantTimeEqStrings => Bool, oid::FUNC_CONSTANT_TIME_EQ_STRINGS_OID;
        },
```

Reply:
Actually, it just occurred to me that these should go in `MZ_CATALOG_BUILTINS` rather than `PG_CATALOG_BUILTINS`, around line 3340.